### DISCUSSION TITLE
why v-memo doesn't work?

### DISCUSSION THREAD

**Discussion by manyuemeiquqi** [vue-macros/vue-macros#605] [Q&A] [answered]
my viteconfig
![image](https://github.com/vue-macros/vue-macros/assets/54520846/a0e4a083-460b-4a03-a41f-243ac43e2b23)
![image](https://github.com/vue-macros/vue-macros/assets/54520846/2958fb39-2515-4d11-a330-fb5323437ced)

bug vue warn
![image](https://github.com/vue-macros/vue-macros/assets/54520846/d707f3f9-9ed9-437f-88f8-3cfac6b35252)

**Comment by zhiyuanzmj** [comment]
@manyuemeiquqi 
Hi, can you provide a minimal reproduction?

**Comment by manyuemeiquqi** [comment]
I attempt optimize jsx performance in this file 
https://github.com/krausest/js-framework-benchmark/blob/master/frameworks/keyed/vue-jsx/src/App.jsx#L115 
like evan you do in 
https://github.com/krausest/js-framework-benchmark/blob/master/frameworks/keyed/vue/src/App.vue#L142， 

but i don't know why memo doesn't work , and i remove it.

I want to ask you, is it only support in .vue file ?

like this ?
```js
script lang="tsx"
``` 
@zhiyuanzmj

**Reply by zhiyuanzmj** [reply, marked as answer]
Include option only support `.setup.tsx` | `.setup.jsx` | `.vue` by default. 
You need to add `.jsx` to include option:
```ts
vueMacros({
  jsxDirective:{
    include: [/\.[jt]sx?$/, /\.vue$/]
  },
  plugins:{
    vue:vue(),
    vueJsx:vueJsx()
  }
}),
```

**Reply by zhiyuanzmj** [reply]
Maybe I should add `.jsx` to include as the default.
